### PR TITLE
Docstring improvement for LineAnimator

### PR DESCRIPTION
**Pull request by DanRyanIrish**
This PR clarifies the new API of ```axis_ranges``` in the ```LineAnimator``` class.

### CODE REVIEW

**Comment by Cadair** [comment]
This is not very readable when rendered: https://57-2165383-gh.circle-artifacts.com/0/home/circleci/project/docs/_build/html/api/sunpy.visualization.imageanimator.LineAnimator.html#sunpy.visualization.imageanimator.LineAnimator

I think it's a little too much info for inside the parameter section, maybe put the detail in `Notes` further down or something?

**Comment by DanRyanIrish** [comment]
Good suggestion @Cadair.  I've made that change.

File: sunpy/visualization/imageanimator.py [hunk 1]
```diff
@@ -827,6 +812,26 @@ class LineAnimator(ArrayAnimator):
 
     Extra keywords are passed to plot.
 
+    Notes
+    -----
+    Additional information on API of axes_ranges kwarg.
+    X-axis values must be supplied (if desired) as an array in the element of
+    the axis_ranges list corresponding to the plot_axis_index in the data array,
+    i.e. x_axis_values == axis_ranges[plot_axis_index].
```
Comment (Cadair) [inline sunpy/visualization/imageanimator.py]: I would make sure the code section in this are in double backticks
```
``
```

File: sunpy/visualization/imageanimator.py [hunk 2]
```diff
@@ -827,6 +812,26 @@ class LineAnimator(ArrayAnimator):
...
+    The shape of the x-axis values array can take two forms.  First, it can equal the
```
Comment (Cadair) [inline sunpy/visualization/imageanimator.py]: maybe enumerate this out as a numbered list? iirc you can do this in rst like:
```
#.
#.
```

**Comment by DanRyanIrish** [comment]
@Cadair, like this?

**Comment by Cadair** [comment]
@DanRyanIrish I think the indentation is wrong maybe?!
```
:1: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
:1: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
:1: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
:1: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
```

**Comment by DanRyanIrish** [comment]
I did 4 space indents for the bullet points which are sub-points of one of the enumerated bullets.  What should the spacing be?

**Comment by Cadair** [comment]
lgtm [here](https://421-2165383-gh.circle-artifacts.com/0/home/circleci/project/docs/_build/html/api/sunpy.visualization.imageanimator.LineAnimator.html#sunpy.visualization.imageanimator.LineAnimator)

**Comment by DanRyanIrish** [comment]
Great!  Travis tests running and passing so far.

**Comment by DanRyanIrish** [comment]
Tests pass!  lgtm.

**Pull request merged**